CHANGELOG.rst

```diff
@@ -19,6 +19,7 @@ Changelog
 
 - Fix conversion of datetime objects to MDS timestamps in the APIs.
 - Also take into account "battery_charged" event type from providers.
+- Added areas methodField to DeviceSerializer
```

Comment:
To remove here. A merge from master after a release put your entry in both version.